src/Client/Resources/keen-io-3_0.php

```diff
@@ -40,22 +40,72 @@
                     'type'        => 'string',
                     'required'    => true,
                 ),
-                'project_data' => array(
+                'name' => array(
+                    'location' => 'json',
+                    'type'     => 'string',
+                    'required' => true,
+                ),
+                'users' => array(
+                    'location' => 'json',
+                    'type'     => 'array',
+                    'required' => true,
+                ),
+                'preferences' => array(
+                    'location' => 'json',
+                    'type'     => 'object',
+                ),
+            ),
+        ),
+
+        'updateProject' => array(
+            'uri'         => 'organizations/{organizationId}/projects/{projectId}',
+            'description' => 'Updates a project for the specified organization.',
+            'httpMethod'  => 'POST',
+            'parameters'  => array(
+                'organizationId'  => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'projectId'  => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'organizationKey' => array(
```

Comment:
Would be cool to see this defined elsewhere and reused. Maybe it gets defined at the top of the file as `$organizationKeyValue => array(...);` or something like that?